cms/src/plugins/string-array/server/services/db-manager-services.js

```diff
@@ -0,0 +1,45 @@
+'use strict';
+
+const _ = require("lodash");
```

Comment:
```suggestion
const { isArray, map, trim, unset } = require("lodash-es");
```
It's recommended to only import what you need instead of the complete library, and using this version of lodash will make it easier.
https://www.npmjs.com/package/lodash-es